testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js

```diff
@@ -21,63 +21,62 @@ import "ui/action_sheet";
+const LIST_ITEM_CLASS = "dx-list-item";
+const LIST_ITEM_ICON_CONTAINER_CLASS = "dx-list-item-icon-container";
+const LIST_ITEM_CONTENT_CLASS = "dx-list-item-content";
+const LIST_ITEM_BEFORE_BAG_CLASS = "dx-list-item-before-bag";
 
-var toSelector = function(cssClass) {
+const toSelector = (cssClass) => {
     return "." + cssClass;
 };
 
-
-var SWITCHABLE_DELETE_READY_CLASS = "dx-list-switchable-delete-ready",
-    SWITCHABLE_MENU_SHIELD_POSITIONING_CLASS = "dx-list-switchable-menu-shield-positioning",
-    SWITCHABLE_DELETE_TOP_SHIELD_CLASS = "dx-list-switchable-delete-top-shield",
-    SWITCHABLE_DELETE_BOTTOM_SHIELD_CLASS = "dx-list-switchable-delete-bottom-shield",
-    SWITCHABLE_MENU_ITEM_SHIELD_POSITIONING_CLASS = "dx-list-switchable-menu-item-shield-positioning",
-    SWITCHABLE_DELETE_ITEM_CONTENT_SHIELD_CLASS = "dx-list-switchable-delete-item-content-shield";
+const SWITCHABLE_DELETE_READY_CLASS = "dx-list-switchable-delete-ready";
+const SWITCHABLE_MENU_SHIELD_POSITIONING_CLASS = "dx-list-switchable-menu-shield-positioning";
+const SWITCHABLE_DELETE_TOP_SHIELD_CLASS = "dx-list-switchable-delete-top-shield";
+const SWITCHABLE_DELETE_BOTTOM_SHIELD_CLASS = "dx-list-switchable-delete-bottom-shield";
+const SWITCHABLE_MENU_ITEM_SHIELD_POSITIONING_CLASS = "dx-list-switchable-menu-item-shield-positioning";
+const SWITCHABLE_DELETE_ITEM_CONTENT_SHIELD_CLASS = "dx-list-switchable-delete-item-content-shield";
 
 QUnit.module("switchable menu decorator", {
-    beforeEach: function() {
-        var testDecorator = SwitchableEditDecorator.inherit({
+    beforeEach: () => {
+        const testDecorator = SwitchableEditDecorator.inherit({
 
             modifyElement: function(config) {
                 this.callBase.apply(this, arguments);
 
-                var $itemElement = $(config.$itemElement);
+                const $itemElement = $(config.$itemElement);
 
-                $itemElement.on("dxpreparetodelete", $.proxy(function(e) {
+                $itemElement.on("dxpreparetodelete", $.proxy((e) => {
                     this._toggleDeleteReady($itemElement);
                 }, this));
             },
 
-            _animateForgetDeleteReady: function() {
+            _animateForgetDeleteReady: () => {
                 return $.when().promise();
             },
 
-            _animatePrepareDeleteReady: function() {
+            _animatePrepareDeleteReady: () => {
                 return $.when().promise();
             }
 
         });
 
         decoratorRegistry.register("menu", "test", testDecorator);
     },
-    afterEach: function() {
+    afterEach: () => {
         delete decoratorRegistry.registry.menu.test;
     }
 });
 
-QUnit.test("positioning should be enabled while item prepared to delete", function(assert) {
-    var $list = $($("#templated-list").dxList({
+QUnit.test("positioning should be enabled while item prepared to delete", (assert) => {
+    const $list = $($("#templated-list").dxList({
```

Comment:
```suggestion
    const $list = $("#templated-list").dxList({
```

I see other `$($(` entries below. Let's remove it.